aioredis/connection.py

```diff
@@ -39,6 +39,7 @@
     'PSUBSCRIBE', b'PSUBSCRIBE',
     'UNSUBSCRIBE', b'UNSUBSCRIBE',
     'PUNSUBSCRIBE', b'PUNSUBSCRIBE',
+    'PING', b'PING',
```

Comment:
Try removing `ping` from `_PUBSUB_COMMANDS`, that is the reason why pool tests are failing